src/widgets/WalletModal/AccountModal.tsx

```diff
@@ -34,7 +34,7 @@ const AccountModal: React.FC<Props> = ({ account, logout, onDismiss = () => null
         variant="secondary"
         onClick={() => {
           logout();
-          window.localStorage.removeItem(localStorageKey);
+          window.localStorage.removeItem(connectorLocalStorageKey);
           onDismiss();
           window.location.reload();
```

Comment:
Just a note - this was put in to fix an issue with logging out. It might not be relevant anymore.

Reply:
which issue ?